pages/Hydra.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: page
+title: Hydra Ecosystem | Hydraecosystem.org
+permalink: /Hydra
+---
+
+# What is Hydra?
+Hydra is a framework that enables REST APIs to be described semantically using RDF. It is based on JSON-LD and proposed as <a href="https://www.hydra-cg.com/spec/latest/core/">W3C draft</a>. It can also be formulated as a documentation framework based on Linked Data which provides vocabulary (similar to schema.org) that aids in the creation of Next Generation Web APIs.
+
+# What's so unique about it?
+
+Hydra in a nutshell is just an extension of JSON-LD technology. As you might already know that our current web is transitioning to Semantic web. In order to acheive that we need to use semantic web tools where ever possible. We can confidently say that Hydra is the most semantically sound tool at the moment when we talk of documentation frameworks. 
+* It is 100% Semantic Web tool as requested by W3C. 
+* It is also directly interoperable with any existing RDF/OWL repository, [(have a look)](https://en.wikipedia.org/wiki/Linked_data#Linking_Open_Data_community_project). 
+* It is easily parsable into less fluent API documentation standards. As RDF is semantically highly descriptive, it is much easier to parse documents into a less rich standard flawlessly. While the opposite, such as parsing an OpenAPI document to make its API interoperable with existing Semantic repositories, can take a long time. 
```

Comment:
what do you mean by less fluent API standards? I don't think any of the things above make sense.